nautobot/extras/tests/test_api.py

```diff
@@ -538,6 +539,27 @@ def test_run_job_with_permission(self):
         self.assertHttpStatus(response, status.HTTP_200_OK)
         self.assertEqual(response.data["result"]["status"]["value"], "pending")
 
+    @skipIf(not rq_worker_running, "RQ worker not running")
+    @override_settings(EXEMPT_VIEW_PERMISSIONS=["*"], JOBS_ROOT=THIS_DIRECTORY)
+    def test_run_job_object_var_lookup(self):
+        self.add_permissions("extras.run_job")
+        d = DeviceRole.objects.create(name="role", slug="role")
+        job_data = {"var4": {"name": "role"}}
```

Comment:
Seems like we also need a test case for `{"var4": d.pk}` as an input, if we're supporting that as well?

Reply:
Right, that makes sense. This behavior was introduced in #693, but it makes sense to introduce the test case now.